github/provider_utils.go

```diff
@@ -55,6 +54,14 @@ func skipUnlessMode(t *testing.T, providerMode string) {
 		} else {
 			t.Log("GITHUB_TOKEN and GITHUB_OWNER environment variables should be set")
 		}
+		// TODO(kfcampbell): this is a problem. how are we going to know it's an organization
+		// in acceptance tests?
+		// ideas:
+		// - we could perform a lookup online to determine the type
```

Comment:
I like this option most given we cache and reuse the result as aggressively as possible.  Debug logs suggest we currently resolve an owner to an organization multiple times.